src/beamlime/dashboard/widgets/plot_creation_widget.py

```diff
@@ -0,0 +1,375 @@
+
+    def _update_dependent_widgets(self) -> None:
+        """Update plot selector based on job and output selection."""
+        if self._selected_job is None:
+            # No job selected - disable everything
+            self._plot_selector.options = []
+            self._plot_selector.value = None
+            self._plot_selector.disabled = True
+            self._create_button.disabled = True
+            return
+
+        # Get available sources for selected job
+        job_data = self._job_service.job_data.get(self._selected_job, {})
+        available_sources = list(job_data.keys())
+
+        if not available_sources:
+            self._plot_selector.options = []
+            self._plot_selector.value = None
+            self._plot_selector.disabled = True
+            self._create_button.disabled = True
+            return
+
+        # Update plot selector
+        try:
+            available_plots = self._plotting_controller.get_available_plotters(
+                self._selected_job, self._selected_output
+            )
+            if available_plots:
+                # Create options with plot class names
+                options = {spec.title: name for name, spec in available_plots.items()}
+                self._plot_selector.options = options
+                self._plot_selector.value = next(iter(options)) if options else None
+                self._plot_selector.disabled = False
+                self._create_button.disabled = False
+            else:
+                self._plot_selector.options = []
+                self._plot_selector.value = None
+                self._plot_selector.disabled = True
+                self._create_button.disabled = True
+        except Exception:
+            self._plot_selector.options = []
+            self._plot_selector.value = None
+            self._plot_selector.disabled = True
+            self._create_button.disabled = True
+
+    def _on_create_plot(self, event) -> None:
+        """Handle create plot button click."""
+        # Validate selection
+        is_valid, errors = self._validate_selection()
+        if not is_valid:
+            return
+
+        # Get available sources
+        job_data = self._job_service.job_data.get(self._selected_job, {})
+        available_sources = list(job_data.keys())
+
+        # Get plot spec
+        plot_name = self._plot_selector.value
+        spec = self._plotting_controller.get_spec(plot_name)
+
+        # Create configuration adapter
+        config = PlotConfigurationAdapter(
+            job_number=self._selected_job,
+            output_name=self._selected_output,
+            plot_spec=spec,
+            available_sources=available_sources,
+            plotting_controller=self._plotting_controller,
+            success_callback=self._on_plot_created,
+        )
+
+        # Create and show configuration modal
+        modal = ConfigurationModal(config=config, start_button_text="Create Plot")
```

Comment:
Will you ever need to access the modal at a later stage? Should it be a class attribute?

Reply:
My plan is to allow for re-opening the modal from a plot tab. But I think we would recreate it based on config, not reuse it. We don't want 20 of them floating about.